checkov/secrets/runner.py

```diff
@@ -458,3 +458,20 @@ def _modify_invalid_secrets_check_result_to_skipped(report: Report) -> None:
                 logging.error(f"Failed to remove suppressed secrets violations from failed_checks, report is corrupted."
                               f"Tried to delete entry {idx} from failed_checks of length {len(report.failed_checks)}",
                               exc_info=True)
+
+    @staticmethod
+    def _get_added_and_removed_commit_hash(
+            key: str, enable_git_history_secret_scan: bool) -> Tuple[str | None, str | None]:
+        """
+        now we have only the current commit_hash - in the added_commit_hash or in the removed_commit_hash.
+        in the next step we will add the connection and the missing data
+        The key is built like this:
+        '{added_commit_hash}_{removed_commit_hash or the string SECRET_NOT_BEEN_REMOVED
+        if the secret not been removed}_{file_name}'
+        """
+        if not enable_git_history_secret_scan:
+            return None, None
+        split_key = key.split('_')
+        added_commit_hash = split_key[0]
+        removed_commit_hash = split_key[1] if split_key[1] != GIT_HISTORY_NOT_BEEN_REMOVED else None
```

Comment:
split_key WILL ALWAYS have at least 2 entries?

Reply:
if ita git history and we get to this function so yes. 
i set the key in the scan_git_history.py line 68 'key = f'{secret_data["added_commit_hash"]}_{removed}_{secret_data["potential_secret"].filename}''